### DISCUSSION TITLE
Time is gone

### DISCUSSION THREAD

**Discussion by Superstrikerz** [fast4x/RiMusic#4378] [General]
I stopped using rimusic since for the past few month Im trying all the methods as  you suggest but still it didnt play songs stating "songs can not played " then i found an alternative its same as rimusic call "YOUTIFY " available on playstore itself 
i not teliing your app is bad 
I know about rimusic as i was using it for a year 
I just want to speak about this

**Comment by fergassss1** [comment]
Then why didnt you open a issue?

**Comment by Superstrikerz** [comment]
Coz they mentioned that if still u cant hear song  then youtube is blocking you try to log in through youtube after that too nothing happened so i didn't open a issue for that
